### PR TITLE
PCHR-1359: Layout changes

### PR DESCRIPTION
**Pull request by danilo-compucorp**
This PR customizes the style of the "create document" modal dialog.

**before**

before

**after**
layout

layout2
(failed validation state)

### CODE REVIEW

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_forms.scss [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+    opacity: 2;
+
+    .ui-select-container.no-border {
+      * {
+        border: none !important;
+        box-shadow: none !important;
+      }
+
+      .ui-select-choices {
+        border: 1px solid #333333 !important;
```
Comment (brunoqueiros) [inline uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_forms.scss]: Use variables for color, the same on line 19, 44

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_forms.scss [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+      .ui-select-toggle {
+        padding: 0;
+
+        .caret::before {
+          position: relative;
+          top: -10px;
+          left: -10px;
+
+          // Magnifying Glass
+          content: "\f002";
```
Comment (brunoqueiros) [inline uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_forms.scss]: Isn't better to use the @fa-icon mixin? I don't know if exist here.

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_forms.scss (hunk 1 above)
Comment (danilo-compucorp) [inline uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_forms.scss]: It was already like that, and soon this bootstrap version will be consolidated with _bootstrapcivicrm_, but it's fixed now

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_forms.scss (hunk 2 above)
Comment (danilo-compucorp) [inline uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_forms.scss]: Same idea as the above answer

**Pull request closed without merging**